test_doc.sh

```diff
@@ -7,6 +7,7 @@ cd ..
 cd chainer
 python setup.py -q build -j 4 develop install --user || python setup.py -q develop install --user
 
+pip install -e .[docs]
```

Comment:
It looks `python -m` should be used like the next line.

Reply:
and the installation of Chainer itself is duplicated with the above line. Can we merge them?

Reply:
I couldn't find a way to install dependencies listed in `extra_require` using `python setup.py ...`. 
I would like to merge the two installation commands. Is it ok to delete `python setup.py ...` on line 8 and use `pip install -e .[docs]` instead?